.github/workflows/build-electron-production.yml

```diff
@@ -26,12 +25,12 @@ jobs:
         env:
           KEYS_DECRYPT_PASSPHRASE: ${{ secrets.KEYS_DECRYPT_PASSPHRASE }}
       - name: Build/release Electron app
-        uses: samuelmeuli/action-electron-builder@v1
+        uses: coparse-inc/action-electron-builder@v1.0.0
         with:
           github_token: ${{ secrets.github_token }}
           args: '-p always'
       - name: Store artifacts
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
for me: Was it broken and do we need this change or is it just to be up-to-date? Quick search v2 vs v3 doesn't give me a lot of insights...

same for `checkout`

Reply:
Just to stay up to date, not sure if there was a deprecation warning yet.